package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "planout",
-  "version": "2.0.2",
+  "version": "3.0.2",
```

Comment:
Fair enough but I don't think that anyone is using PlanOut cross platform such that they actually need backwards compatibility with the old way of specifying compatability.